modules/rosa-sts-setting-up-environment.adoc

```diff
@@ -201,31 +201,6 @@ $ rosa download openshift-client
 ----
 $ rosa verify openshift-client
 ----
-. Optional: If you use a permissions boundary policy, gather the permissions boundary policy Amazon resource name (ARN) from your AWS account.
```

Comment:
Will you please double-check that this content should be removed? It scans like it's out of scope for this PR.

Reply:
As far as I can tell, the point of this step is to tee up an environment variable in a terminal window that is later referenced in the procedures in `modules/rosa-sts-creating-roles.adoc`. From a review of the topic map in the `dedicated-4` branch, I don't think that module is actually published in the docs. According to the `./scripts/list_assemblies_used_by_modules.sh` script, that module is only used in the following assembly:

```
$ ./scripts/list_assemblies_used_by_modules.sh 
...
Listing the assemblies that include modules/rosa-sts-creating-roles.adoc...

###modules/rosa-sts-creating-roles.adoc
// Module included in the following assemblies:
//
// * rosa_getting_started_sts/rosa-sts-creating-roles.adoc
```
I can't see a reference to that assembly in the topic map:

```
$ grep -is 'rosa-sts-creating-roles' _topic_map.yml 
$ 
```
Also, https://docs.openshift.com/rosa/rosa_getting_started_sts/rosa-sts-creating-roles.html produces a 404 error.

In any case, I have replaced the `${}` style of environment variable with the angle bracket format in `rosa_getting_started_sts/rosa-sts-creating-roles.adoc` as part of this PR, in case it is included in the publication in the future.

Reply:
It might also be useful to add that other than `modules/rosa-sts-setting-up-environment.adoc`, `rosa_getting_started_sts/rosa-sts-creating-roles.adoc` looks to be the only file that includes the string `permissions_boundary_arn` in the `dedicated-4` branch:

```
$ pwd
/home/pneedle/data/github-repos/openshift-docs

$ git branch | grep -is '^*'
* dedicated-4

$ grep -ris 'permissions_boundary_arn' * --exclude-dir=_preview/
modules/rosa-sts-creating-roles.adoc:    --permissions-boundary ${permissions_boundary_arn}
modules/rosa-sts-creating-roles.adoc:    --permissions-boundary ${permissions_boundary_arn}
modules/rosa-sts-creating-roles.adoc:    --permissions-boundary ${permissions_boundary_arn}
modules/rosa-sts-creating-roles.adoc:    --permissions-boundary ${permissions_boundary_arn}
modules/rosa-sts-setting-up-environment.adoc:$ export permissions_boundary_arn=<ARN from AWS account>
```

Reply:
The same applies to the other steps in this section that I removed (steps 9 to 11). It looks like the purpose of those steps is to prepare environment variables in the current shell. But, the PR changes the references to those environment variables in the subsequent procedures to be angle bracket replaceables instead. As such, I don't think these preparatory steps are required.